code/modules/admin/topic.dm

```diff
@@ -4077,17 +4077,22 @@
 						log_admin("[key_name_admin(usr)] triggered a FAKE revolution alert.")
 						return
 					//TODO (UPHEAVAL PART 2) think of fake alerts too
-			if("fakebooms") //Micheal Bay is in the house !
+			if("fakebooms") //Michael Bay is in the house !
 				feedback_inc("admin_secrets_fun_used",1)
 				feedback_add_details("admin_secrets_fun_used","FAKEE")
-				var/choice = input("How much high-budget explosions do you want ?", "Micheal Bay SFX Systems", 1) as num
-				if(choice < 1) //No negative or null explosion amounts here math genius
+				var/amount = input("How much high-budget explosions do you want ?", "Michael Bay SFX Systems", 1) as num
+				if(amount < 1) //No negative or null explosion amounts here math genius
 					to_chat(usr, "<span class='warning'>Invalid input range (null or negative)</span>")
 					return
-				message_admins("[key_name_admin(usr)] improvised himself as Micheal Bay and triggered [round(choice)] fake explosions.")
-				log_admin("[key_name_admin(usr)] improvised himself as Micheal Bay and triggered [round(choice)] fake explosions.")
-				for(var/i = 1 to choice)
-					world << sound('sound/effects/explosionfar.ogg')
+				var/realeffect = alert(usr,"Use visible explosions?", "Michael Bay SFX Systems", "Yes", "No") == "Yes"
+				message_admins("[key_name_admin(usr)] improvised himself as Michael Bay and triggered [round(amount)] fake explosions.")
+				log_admin("[key_name_admin(usr)] improvised himself as Michael Bay and triggered [round(amount)] fake explosions.")
```

Comment:
The wording on this is so fucking bad.

Reply:
Which part? My alert or the logging? Because I didn't write the logs.

Reply:
The logging has bad word choice "improvised" instead of "imitated" + bad grammar

Just delete the dumb Bay reference

Reply:
Done.

Reply:
Man, someone thought Michael Bay jokes were really fucking funny